packages/react-instantsearch-hooks-dom/src/widgets/__tests__/RefinementList.test.tsx

```diff
@@ -0,0 +1,3462 @@
+                  type="checkbox"
+                  value="Sony"
+                />
+                <span
+                  class="ais-RefinementList-labelText"
+                >
+                  SONY
+                </span>
+                <span
+                  class="ais-RefinementList-count"
+                >
+                  350
+                </span>
+              </label>
+            </li>
+            <li
+              class="ais-RefinementList-item"
+            >
+              <label
+                class="ais-RefinementList-label"
+              >
+                <input
+                  class="ais-RefinementList-checkbox"
+                  type="checkbox"
+                  value="Incipio"
+                />
+                <span
+                  class="ais-RefinementList-labelText"
+                >
+                  INCIPIO
+                </span>
+                <span
+                  class="ais-RefinementList-count"
+                >
+                  320
+                </span>
+              </label>
+            </li>
+            <li
+              class="ais-RefinementList-item"
+            >
+              <label
+                class="ais-RefinementList-label"
+              >
+                <input
+                  class="ais-RefinementList-checkbox"
+                  type="checkbox"
+                  value="KitchenAid"
+                />
+                <span
+                  class="ais-RefinementList-labelText"
+                >
+                  KITCHENAID
+                </span>
+                <span
+                  class="ais-RefinementList-count"
+                >
+                  318
+                </span>
+              </label>
+            </li>
+            <li
+              class="ais-RefinementList-item"
+            >
+              <label
+                class="ais-RefinementList-label"
+              >
+                <input
+                  class="ais-RefinementList-checkbox"
+                  type="checkbox"
+                  value="Whirlpool"
+                />
+                <span
+                  class="ais-RefinementList-labelText"
+                >
+                  WHIRLPOOL
+                </span>
+                <span
+                  class="ais-RefinementList-count"
+                >
+                  298
+                </span>
+              </label>
+            </li>
+          </ul>
+        </div>
+      </div>
+    `);
+  });
+
+  test('forwards `div` props to the root element', async () => {
+    const client = createSearchClient({ search });
+    const { container } = render(
+      <InstantSearchHooksTestWrapper searchClient={client}>
+        <RefinementList
+          attribute="brand"
+          className="MyRefinementList"
+          title="Some custom title"
+        />
+      </InstantSearchHooksTestWrapper>
+    );
+
+    await wait(0);
+
+    expect(container).toMatchInlineSnapshot(`
```

Comment:
I don't think we need this snapshot. We can use `toHaveAttribute()` on `container.firstChild`.